lib/LMSManagers/LMSCustomerManager.php

```diff
@@ -824,6 +829,16 @@ public function GetCustomer($id, $short = false)
                     if ($types)
                         $result['contacts'][$idx]['typestr'] = implode('/', $types);
                 }
+            if (is_array($result['emails']))
+                foreach ($result['emails'] as $idx => $row) {
+                    $types = array();
+                    foreach ($CONTACTTYPES as $tidx => $tname)
+                        if ($row['type'] & $tidx)
+                            $types[] = $tname;
+
+                    if ($types)
+                        $result['emails'][$idx]['typestr'] = implode('/', $types);
+                }
```

Comment:
To ma na celu dodanie wyświetlania opisu przy adresie email tak jak ma to miejsce przy numerze telefonu
![image](https://cloud.githubusercontent.com/assets/15612188/10998030/56f7379a-8492-11e5-8cae-de5f3fbf6655.png)